### PR TITLE
Add cell_number_type setting

### LINKED ISSUE #12: Cell number mismatch?
I found the cell number displayed in the notification message does not match the cell number of the notebook.
It looks that the `notebook.activeCellIndex` returns the position and does not care about the number of execution.

See the following image (captured from the readme page):

cell-number-mismatch

### PR DESCRIPTION
**Pull request by i-aki-y**
To fix #12, I replaced `notebook.activeCellIndex` for `executionCount`.

See, https://jupyterlab.readthedocs.io/en/stable/api/classes/cells.codecellmodel-1.html#executioncount

I would appreciate it if you could look at my PR.

### CODE REVIEW

**Review by mwakaba2 — COMMENTED** [review]
Thanks for the PR! I'd still think `activeCellIndex` would be useful to have.  
How about creating a setting called `report_cell_execution_count` and making that configurable?

**Comment by i-aki-y** [comment]
@mwakaba2  Hi, I revised the PR.
As discussed in #12, I added a new setting, `cell_number_type`, that allows us to select how the cell number is displayed.
I would appreciate it if you review this PR.

File: README.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ Use the following settings to update cell execution time for a notification and
     // Settings for the Notifications extension
     // ****************************************
 
+    // Cell Number Type
+    // Type of cell number to display when the report_cell_number is true. Select from 'cell_index' or ‘cell_execution_count'.
+    "cell_number_type": "cell_index",
+
```
Comment (mwakaba2) [inline README.md]: I'd add this below the `report_cell_number` to match with the config. Also for the user reading this, they'll have more context on what a `report_cell_number` is before reading `cell_number_type`.

**Review by mwakaba2 — APPROVED** [review]
I made one small comment, but everything looks good! Thanks for updating this!

File: README.md (hunk 1 above)
Comment (i-aki-y) [reply README.md]: @mwakaba2 Thank you for your comment. 
Yes, I agree with you. But I have some difficulty.
These setting examples are copied from the actual default settings of my development version.
It seems that the setting entries are automatically reordered by alphabetic order (I also saw the other settings.)
So, I can move the position of `cell_number_type` just for README.md, but I can not move the actual entry position.

If it's OK just to fix the README, I'd like to fix it. Can I ask what you think about it?

**Comment by mwakaba2** [comment]
I see, interesting. Just updating the README works for me! 

On May 13, 2021, Mariko Wakabayashi ***@***.***> wrote:
> @i-aki-y commented on this pull request.
>

> In README.md <https://github.com/mwakaba2/jupyterlab-
> notifications/pull/13#discussion_r632220837>:
>

> @@ -42,6 +42,10 @@ Use the following settings to update cell execution
time for a notification and // Settings for the Notifications extension
// **************************************** + // Cell Number Type + //
Type of cell number to display when the report_cell_number is true.
Select from 'cell_index' or ‘cell_execution_count'. +
"cell_number_type": "cell_index", +
> @mwakaba2 <https://github.com/mwakaba2> Thank you for your comment.
>  Yes, I agree with you. But I have some difficulty.
>  These setting examples are copied from the actual default settings of
> my development version.
>  It seems that the setting entries are automatically reordered by
> alphabetic order (I also saw the other settings.)
>  So, I can move the position of cell_number_type just for README.md,
> but I can not move the actual entry position.
>
> If it's OK just to fix the README, I'd like to fix it. Can I ask what
> you think about it?
>

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/mwakaba2/jupyterlab-
notifications/pull/13#discussion_r632220837>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/AA2VZMPMASMCQXJCFPDLW6DTNSAAPANCNFSM44QEI7JQ>.

**Comment by i-aki-y** [comment]
@mwakaba2  I fixed README.md.

**Comment by mwakaba2** [comment]
@i-aki-y awesome! thanks so much for the contribution. I'll merge and make a release before end of day today. (Pacific Time)

**Pull request merged**